### PR TITLE
clarify who can be assigned GPU for spaces

### PR DESCRIPTION
**Pull request by AK391**
update spaces docs to clarify that individuals with spaces repos can also request to be assigned a gpu

### CODE REVIEW

File: docs/hub/spaces.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The default Spaces environment comes with several pre-installed dependencies:
 
 * [`datasets`](https://github.com/huggingface/datasets) allows you to fetch or display datasets from inside your app easily.
 
-Each Spaces environment is limited to 16GB RAM and 8 CPU cores. Organization [subscribers](https://huggingface.co/pricing) (Lab, Startup, and Enterprise) can access Spaces with one T4 GPU on a case-by-case basis. Please email us at **website at huggingface.co** or let us know on [Twitter](https://twitter.com/huggingface) if you need one.
+Each Spaces environment is limited to 16GB RAM and 8 CPU cores. Individuals and Organization [subscribers](https://huggingface.co/pricing) (Lab, Startup, and Enterprise) can access Spaces with one T4 GPU on a case-by-case basis. Please email us at **website at huggingface.co** or let us know on [Twitter](https://twitter.com/huggingface) if you need one.
```
Comment (osanseviero) [inline docs/hub/spaces.md:27]:
```suggestion
Each Spaces environment is limited to 16GB RAM and 8 CPU cores. Individuals and Organization [subscribers](https://huggingface.co/pricing) (Lab, Startup, and Enterprise) can access Spaces with one T4 GPU on a case-by-case basis. Please email us at **website at huggingface.co**.
```

**Review by osanseviero — APPROVED** [review]
Thanks for the PR! I left a minor comment and we can merge afterwards :rocket:

**Comment by AK391** [comment]
> Thanks for the PR! I left a minor comment and we can merge afterwards 🚀

thanks, updated

**Comment by osanseviero** [comment]
Awesome :fire:

**Pull request merged**